src/firefly/js/visualize/ui/ColorDialog.jsx

```diff
@@ -132,7 +132,7 @@ function renderStandardThreeColorView(plot,rFields,gFields,bFields) {
     return (
         <Box sx={{pt:.5}}>
             <FieldGroup groupKey={'colorDialogTabs'} keepState={false}>
-                <FieldGroupTabs initialState= {{ value:'red' }} fieldKey='colorTabs'>
+                <FieldGroupTabs initialState= {{ value:'red' }} tabId='colorTabs' fieldKey='colorTabs'>
```

Comment:
We can make `tabId` in TabPanel default to a unique ID if not given.

Reply:
it creates a unique id, that was the problem. It kept re-rendering and creating more ids

Reply:
I see.  In this case, maybe `FieldGroupTabs` should internally set tabId to fieldKey.